ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1717,6 +1717,26 @@
       }
     ]
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseFieldID": "takenOfflineByStaffDate",
```

Comment:
why is this needed?

Reply:
needed to separate the taking offline by user/system